stable/fairwinds-insights/ci/test-values.yaml

```diff
@@ -48,6 +48,7 @@ emailCronjob:
   schedules: []
 
 databaseCleanupCronjob:
+  enabled: true
```

Comment:
actually just to think of it ... you do not need this. none of the cronjob are enabled here